adaptor.js

```diff
@@ -482,6 +482,64 @@ function convertToApis(source,obj,defaults) {
     return apis;
 }
 
+function convertToPaths(source,obj,defaults) {
+    let paths = [];
+    for (let p in source.paths) {
+        for (let m in source.paths[p]) {
+            if ((m !== 'parameters') && (m !== 'summary') && (m !== 'description') && (!m.startsWith('x-'))) {
+                let op = source.paths[p][m];
+                let tagName = 'Default';
+                if (op.tags && op.tags.length > 0) {
+                    tagName = op.tags[0];
+                }
+                let entry = paths.find(function(e,i,a){
+                    return (e.name === p);
+                });
+                if (!entry) {
+                    const split = p.replace(/^\//,'').split(/\//g);
+                    const dirname = split.slice(0,-1).join('/');
+                    const filename = split.slice(-1).join('');
+                    const className = split.map(v=>v.replace(/{([^}]+)}/g,(v,v1)=>`By${v1[0].toUpperCase()}${v1.slice(1)}`).replace(/^./,(v)=>`${v[0].toUpperCase()}${v.slice(1)}`)).join('')
```

Comment:
Can we have a comment on what this map / regex is doing? Is it dependent on the path value having a particular format? Also missing semi-colon.

Reply:
Added comments